src/Diagnostics.RuntimeHost/Controllers/ApiManagement/ApiManagementController.cs

```diff
@@ -28,15 +28,15 @@ public async Task<IActionResult> ExecuteQuery(string subscriptionId, string reso
         }
 
         [HttpPost(UriElements.Detectors)]
-        public async Task<IActionResult> ListDetectors(string subscriptionId, string resourceGroupName, string serviceName, [FromBody] dynamic postBody, [FromQuery(Name = "text")] string text = null)
+        public async Task<IActionResult> ListDetectors(string subscriptionId, string resourceGroupName, string serviceName, [FromBody] dynamic postBody, [FromQuery(Name = "text")] string text = null, [FromQuery] string l = "")
```

Comment:
nitpick : instead of 'l', can we call it 'locale'

Reply:
I thought about this and then I found in all the ARM calls, they are using something like "**l=ja.ja-jp**". For example:
 
https://appservice-diagnostics.azurefd.net/?clientOptimizations=undefined&**l=ja.ja-jp**&trustedAuthority=https%3A%2F%2Fms.portal.azure.com&shellVersion=undefined

I am open to modify this if that's more understandable:)

Reply:
ah ok.. nvm then

Reply:
LOL I was thinking the same thing